extensions/ipynb/.vscodeignore

```diff
@@ -1,6 +1,8 @@
 .vscode/**
 src/**
+notebook-src/**
 out/**
+notebook-out/**
```

Comment:
I think `notebook-out` needs to be included since it contains the compiled js. Only the directory with the typescript should be excluded

Reply:
hm, everything builds and runs without error with this version of the `.vscodeignore`, and `/src` includes all of the primary .js files and is ignored as well. `src/**` was in the `.vscodeignore` from before I started working in the extension, should I remove that as well?